client/vscode/src/inlineCompletionActivation.ts

```diff
@@ -6,40 +6,76 @@ import {
+    { scheme: 'file', language: 'scala' },
+    { scheme: 'file', language: 'vue' },
+]
 
 export function registerInlineCompletion(languageClient: LanguageClient) {
     const inlineCompletionProvider = new CodeWhispererInlineCompletionItemProvider(languageClient)
-    languages.registerInlineCompletionItemProvider(
-        [
-            { scheme: 'file', language: 'typescript' },
-            { scheme: 'file', language: 'json' },
-            { scheme: 'file', language: 'yaml' },
-            { scheme: 'file', language: 'java' },
-            { scheme: 'file', language: 'go' },
-            { scheme: 'file', language: 'php' },
-            { scheme: 'file', language: 'rust' },
-            { scheme: 'file', language: 'kotlin' },
-            { scheme: 'file', language: 'terraform' },
-            { scheme: 'file', language: 'ruby' },
-            { scheme: 'file', language: 'shellscript' },
-            { scheme: 'file', language: 'dart' },
-            { scheme: 'file', language: 'lua' },
-            { scheme: 'file', language: 'powershell' },
-            { scheme: 'file', language: 'r' },
-            { scheme: 'file', language: 'swift' },
-            { scheme: 'file', language: 'systemverilog' },
-            { scheme: 'file', language: 'scala' },
-            { scheme: 'file', language: 'vue' },
-        ],
-        inlineCompletionProvider
-    )
+    languages.registerInlineCompletionItemProvider(CodewhispererInlineCompletionLanguages, inlineCompletionProvider)
+
+    commands.registerCommand('aws.sample-vscode-ext-amazonq.invokeInlineCompletion', async (...args: any) => {
+        // Register manual trigger command for InlineCompletions
+        await commands.executeCommand(`editor.action.inlineSuggest.trigger`)
+    })
+
+    // Simple implementation of logInlineCompletionSessionResultsNotification
+    const onInlineAcceptance = async (
+        sessionId: string,
+        itemId: string,
+        requestStartTime: number,
+        firstCompletionDisplayLatency?: number
+    ) => {
+        console.log('OnInlineAcceptance called with: ', sessionId, itemId)
+        const params: LogInlineCompletionSessionResultsParams = {
+            sessionId: sessionId,
+            completionSessionResult: {
+                [itemId]: {
+                    seen: true,
+                    accepted: true,
+                    discarded: false,
+                },
+            },
+            totalSessionDisplayTime: Date.now() - requestStartTime,
+            firstCompletionDisplayLatency: firstCompletionDisplayLatency,
+        }
+        languageClient.sendNotification(logInlineCompletionSessionResultsNotificationType, params)
+    }
+    commands.registerCommand('aws.sample-vscode-ext-amazonq.accept', onInlineAcceptance)
 }
 
-class CodeWhispererInlineCompletionItemProvider implements InlineCompletionItemProvider {
+export class CodeWhispererInlineCompletionItemProvider implements InlineCompletionItemProvider {
```

Comment:
is the export here intentional?

Reply:
forgot to remove it, but it's really minor since this is test extension